tech-ally/src/app/services/local-storage.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalStorageService {
+  localStorage: Storage;
+
+  constructor() {
+    this.localStorage = window.localStorage;
+  }
+
+  get(key: string): any {
+    if (this.isLocalStorageSupported) {
+      let res: string = this.localStorage.getItem(key) ?? "";
+      return JSON.parse(res)
+    }
+    return null;
+  }
+
+  set(key: string, value: any): boolean {
+    if (this.isLocalStorageSupported) {
+      this.localStorage.setItem(key, JSON.stringify(value));
+      return true;
+    }
+    return false;
+  }
+
+  remove(key: string): boolean {
```

Comment:
I don't see this function being used... 🤔

Reply:
Currently it's not. I'd remove it but I'm sure we'll certainly need to clear local storage. I'd prefer to leave it for now.